### PR TITLE
Initial GitHub submission

### PR DESCRIPTION
**Pull request by joepound**
- Project designed as per objectives
- Media query styling included as per stretch goal

### CODE REVIEW

File: great-idea/services.html [hunk 1]
```diff
@@ -1,107 +1,93 @@
...
+                    <a class="logo-link" title="Great Idea! Company logo." href="index.html"></a>
+                </nav>
+                <img class="header-image" src="http://i63.tinypic.com/u9d38.jpg" alt="Our services header image">
+                <section class="header-description">
+                    <h1>Services</h1>
+                    <p>Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p>
+                </section>
+            </header>
+            <div class="services">
+                <article>
```
Comment (VivaCode) [inline great-idea/services.html]: Great use of article tags! It shows you understand the difference between an article and a section.

**Comment by joepound** [comment]
Wow, I'm so glad that you went over all my code - it's so helpful to see
comments in several blocks of code and I really appreciate it. :) Thanks,
Emily!

First off, is there a smoother way for me to reply to comments besides
Gmail (or for using GitHub to reply)? I'm new to this feature of GitHub,
and I was thinking about somehow placing my replies to comments on the same
block as your comments for everyone's convenience (instead of me lumping it
all inside a single email). :)

   - Yeah, I am really happy with being able to use percentage values
   because of them being dynamic. Haven't use rem and em before (before your
   comment, I thought that px was the standard for static values in size
   properties), so I'm glad you recommended the other units.
   - Wow, I'm glad that commas have this precedence in CSS selectors - I
   used it the way I did in my current code because I thought that the
   one-liner you gave me would be equivalent to:
       .container > header > .header-description > h1 {}
       h2 {}
       h3 {}
   I now know that commas are more convenient than I thought. :)
   - I actually planned on commenting my code, but I didn't think I'd be
   able to finish thinking on it before the start of the next day. I'll
   definitely add  comments to future commits - I plan to practice on this
   week's projects over the weekend (refactoring, optimization,
   experimentation, new features and other stuff).
   - Thanks for the bit about placing image links outside of navs. Didn't
   expect that what I did wasn't exactly best practice. :)

I'll be setting up repos for previous projects as well (won't do much
committing on Tuesday's project though since Wednesday's project was
essentially an update to that). Thanks again for all your input, Emily!

P.S.: I love that Git pun (intended or not) about "push(ing) to make
yourself better." :D

File: great-idea/services.html [hunk 2]
```diff
@@ -1,107 +1,93 @@
...
+        <div class="container">
+            <header>
+                <nav>
+                    <a class="main-link" href="#">Services</a>
+                    <a class="main-link" href="#">Product</a>
+                    <a class="main-link" href="#">Vision</a>
+                    <a class="main-link" href="#">Features</a>
+                    <a class="main-link" href="#">About</a>
+                    <a class="main-link" href="#">Contact</a>
+                    <a class="logo-link" title="Great Idea! Company logo." href="index.html"></a>
```
[the remark this answers is not in the document]
Comment (joepound) [reply great-idea/services.html]: Thanks for this! That's something new for me - didn't expect that what I did wasn't exactly best practice. :)

File: great-idea/css/style.css [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+.container > header > .header-description > h1,
+.container > header > .header-description > h2,
+.container > header > .header-description > h3 {
+    font-size: 37.5px;
+}
+
+.header-description > p {
+    font-size: 20px;
+}
+
```
[the remark this answers is not in the document]
Comment (joepound) [reply great-idea/css/style.css]: I actually planned on commenting my code, but I didn't think I'd be able to finish thinking on it before the start of the next day. I'll definitely add  comments to future commits - I plan to practice on this week's projects over the weekend (refactoring, optimization, experimentation, new features and other stuff). :)

**Pull request closed without merging**